Calibration/IsolatedParticles/plugins/IsolatedTracksHcalScale.cc

```diff
@@ -1,43 +1,187 @@
-
-#include "DataFormats/HcalDetId/interface/HcalSubdetector.h"
-#include "DataFormats/TrackReco/interface/TrackBase.h"
-#include "Calibration/IsolatedParticles/plugins/IsolatedTracksHcalScale.h"
-
+// system include files
+#include <memory>
```

Comment:
no need to include this if you do not use smart pointers

Reply:
Done